### ISSUE TITLE
VSCode downloads wakatime-cli standalone everytime

### ISSUE THREAD

**Issue by BillChen2K** [wakatime/vscode-wakatime#144]
Hi, I'm using VSCode wakatime plugin in China, and VSCode always get stuck in `Initializing Wakatime`. Downloading the wakatime-cli standalone seems to take too much time(about 30 min) everytime I startup vscode.

Using a global proxy did help, but is it possible for wakatime to cache the wakatime-cli standalone so that it won't redownload it every time?

```
[Extension Host] [WakaTime][DEBUG] ::WakaTime debug mode::
workbench.desktop.main.js:sourcemap:266 [Extension Host] [WakaTime][DEBUG] Initializing WakaTime v4.0.0
workbench.desktop.main.js:sourcemap:266 [Extension Host] [WakaTime][DEBUG] Using standalone wakatime-cli.
workbench.desktop.main.js:sourcemap:266 [Extension Host] [WakaTime][DEBUG] Downloading wakatime-cli standalone...
```

**Comment by alanhamlett** [comment]
It shouldn't be downloading wakatime-cli every time... it should only download when the version changes. On my machine the debug logs look like:

```
[Extension Host] [WakaTime][DEBUG] ::WakaTime debug mode::
[Extension Host] [WakaTime][DEBUG] Initializing WakaTime v4.0.0
[Extension Host] [WakaTime][DEBUG] Using standalone wakatime-cli.
[Extension Host] [WakaTime][DEBUG] Current wakatime-cli version is 13.0.7
[Extension Host] [WakaTime][DEBUG] Checking for updates to wakatime-cli...
[Extension Host] [WakaTime][DEBUG] wakatime-cli is up to date
[Extension Host] [WakaTime][DEBUG] WakaTime: Initialized
```

One of these methods must be triggering the download:
[isStandaloneCliInstalled](https://github.com/wakatime/vscode-wakatime/blob/18fdda1e38fb4a04d2a4550c282cc10ad4a09349/src/dependencies.ts#L88)
[isStandaloneCliLatest](https://github.com/wakatime/vscode-wakatime/blob/18fdda1e38fb4a04d2a4550c282cc10ad4a09349/src/dependencies.ts#L185)

First step is adding debug logs to [this method](https://github.com/wakatime/vscode-wakatime/blob/18fdda1e38fb4a04d2a4550c282cc10ad4a09349/src/dependencies.ts#L106) to find if the file exists check or the version check is failing.

**Comment by iamtodor** [comment]
I have the same issue. 

If I add the following proxy: https://github.com/wakatime/vscode-wakatime/issues/140#issuecomment-666146388
I get the following warnings:
```
[Extension Host] [WakaTime][DEBUG] Initializing WakaTime v4.0.8
console.ts:137 [Extension Host] [WakaTime][DEBUG] Using standalone wakatime-cli.
console.ts:137 [Extension Host] [WakaTime][DEBUG] Downloading wakatime-cli standalone...
console.ts:137 [Extension Host] [WakaTime][WARN] Failed to download https://wakatime-cli.s3-us-west-2.amazonaws.com/mac-x86-64/wakatime-cli.zip
t.log @ console.ts:70
console.ts:137 [Extension Host] [WakaTime][WARN] Error: tunneling socket could not be established, cause=connect ECONNREFUSED 127.0.0.1:1080
t.log @ console.ts:70
console.ts:137 [Extension Host] [WakaTime][DEBUG] WakaTime: Initialized
console.ts:137 [Extension Host] [WakaTime][DEBUG] Status bar icon enabled.
console.ts:137 [Extension Host] Congratulations, your extension "text-compact" is now active!
```

Debug logs without proxy:
```

console.ts:137 [Extension Host] [WakaTime][DEBUG] Initializing WakaTime v4.0.8
console.ts:137 [Extension Host] [WakaTime][DEBUG] Using standalone wakatime-cli.
console.ts:137 [Extension Host] [WakaTime][DEBUG] Downloading wakatime-cli standalone...
console.ts:137 [Extension Host] Congratulations, your extension "text-compact" is now active!
log.ts:197   ERR No application in the Launch Services database matches the input criteria.: Error: No application in the Launch Services database matches the input criteria.
console.ts:137 [Extension Host] [WakaTime][DEBUG] Extracting wakatime-cli into "/Users/todor/.vscode/extensions/wakatime.vscode-wakatime-4.0.8"...
console.ts:137 [Extension Host] [WakaTime][DEBUG] Finished extracting wakatime-cli standalone.
log.ts:197   ERR No application in the Launch Services database matches the input criteria.: Error: No application in the Launch Services database matches the input criteria.
```

The system:
```
OS: 10.15.6
Python3: 3.8.3
VS Code: 1.49.1
Wakatime: v4.0.8
OpenSSL: LibreSSL 2.8.3
```

**Comment by gandarez** [comment]
Try adding these lines to your `~/.wakatime.cfg` file:

```
standalone = false
alpha = true
```

Then restart VS Code. Does that fix it?

**Comment by alanhamlett** [comment]
Might be fixed in v7.0.0 of the vscode extension, now that we're using the new [wakatime-cli](https://github.com/wakatime/wakatime-cli) that downloads from GH Releases instead of S3.
